src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/ReportedAdjudicationController.kt

```diff
@@ -50,4 +54,15 @@ class ReportedAdjudicationController {
   ): Page<ReportedAdjudicationDto> {
     return reportedAdjudicationService.getMyReportedAdjudications(agencyId, pageable)
   }
+
+  @PostMapping(value = ["/{adjudicationNumber}/create-draft-adjudication"])
```

Comment:
Were we going to call them something other than drafts? Actually now I say it - I guess if they are actually drafts then that seems like a reasonable name